tests/assertions.js

```diff
@@ -12,6 +12,26 @@ function shouldThrowInvalidInput({result, error}) {
     'Expected status code 400 invalid input!');
 }
 
+function testIssuedVc({issuedVc}) {
+  issuedVc.should.be.an('object');
+  issuedVc.should.have.property('@context');
+  issuedVc.should.have.property('type');
+  issuedVc.type.should.eql(['VerifiableCredential']);
```

Comment:
this is ok, but Vcs usually have multiple types. might be best to assume one type is `VerifiableCredential`.

Reply:
Updated [15cd05d](https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/7/commits/15cd05d2f671f9b456ac3435244b19100499de51)